### PR TITLE
Fix Symbol to Object comparison

### PR DESCRIPTION
**Pull request by dbatyai**

### CODE REVIEW

**Review by rerobika — APPROVED** [review]
LGTM

**Comment by ossy-szeged** [comment]
LGTM (informal)

File: jerry-core/ecma/operations/ecma-comparison.c [hunk 1]
```diff
@@ -203,7 +203,22 @@ ecma_op_abstract_equality_compare (ecma_value_t x, /**< first operand */
 #if ENABLED (JERRY_ESNEXT)
   if (JERRY_UNLIKELY (ecma_is_value_symbol (x)))
   {
-    return ECMA_VALUE_FALSE;
+    if (!ecma_is_value_object (y))
```
Comment (zherczeg) [inline jerry-core/ecma/operations/ecma-comparison.c:206]: The other cases simply swapped the values.

**Review by zherczeg — CHANGES_REQUESTED** [review]

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**